Assets/HoloToolkit/UI/Scripts/UsabilityUtilities.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A helper class for making applications more usable across different devices.
+    /// </summary>
+    public static class UsabilityUtilities
```

Comment:
I like it!  It's also nice that it's asymptotic, so devices with very high PPDs won't scale content down too fast.  Check out the new changes.